tests/lib/mock.py

```diff
@@ -33,17 +36,16 @@ class MockCVPDatabase:
     CONTAINER_KEY = 'container_1234abcd-1234-abcd-12ab-123456abcdef'
 
     # Data used to initiate the mock database
-    CVP_DATA_CONTAINERS_INIT = {
-        'Tenant': {
-            'name': 'Tenant',
-            'key': 'root',
-            'parentContainerId': None}
-    }
-
-    def __init__(self, devices: dict = None, containers: dict = None, configlets: dict = None):
+    # CVP_DATA_DEVICE_INIT = magickmock_data.MOCKDATA_DEVICES
+    # CVP_DATA_CONTAINERS_INIT = magickmock_data.MOCKDATA_CONTAINERS
+    # CVP_DATA_CONFIGLETS_MAPPERS_INIT = magickmock_data.MOCKDATA_CONFIGLET_MAPPERS
+    # CVP_DATA_CONFIGLET_INIT = magickmock_data.MOCKDATA_CONFIGLETS
+
+    def __init__(self, devices: dict = None, containers: list = None, configlets: list = None, configlets_mappers: dict = None):
```

Comment:
I think types are still dict for containers and configlets

Reply:
If we consider to use output from CV APIs in mock, we should consider to use list as all these outputs are not dict but list.

Reply:
We can cover that in a dedicated PR related to testing and mock